spark_pipeline_framework/transformers/fhir_sender/v2/fhir_sender_processor.py

```diff
@@ -359,6 +359,7 @@ async def send_partition_to_server_async(
                                 log_level=parameters.log_level,
                                 retry_count=parameters.retry_count,
                                 exclude_status_codes_from_retry=parameters.exclude_status_codes_from_retry,
+                                smart_merge=parameters.smart_merge,
```

Comment:
we also need to add this `smart_merge=parameters.smart_merge,` to the same method call in lines 387-405 ?